airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle

```diff
@@ -40,27 +40,25 @@ tasks.register('generate').configure {
 
 dependencies {
 
-    implementation project(':airbyte-cdk:java:airbyte-cdk:dependencies')
-    implementation project(':airbyte-cdk:java:airbyte-cdk:core')
+    implementation project(':airbyte-cdk:java:airbyte-cdk:airbyte-cdk-dependencies')
+    implementation project(':airbyte-cdk:java:airbyte-cdk:airbyte-cdk-core')
 
     implementation 'io.debezium:debezium-api:2.4.0.Final'
     implementation 'io.debezium:debezium-embedded:2.4.0.Final'
     implementation 'org.codehaus.plexus:plexus-utils:4.0.0'
 
-    testFixturesImplementation project(':airbyte-cdk:java:airbyte-cdk:dependencies')
-    testFixturesImplementation testFixtures(project(':airbyte-cdk:java:airbyte-cdk:dependencies'))
-    testFixturesImplementation project(':airbyte-cdk:java:airbyte-cdk:core')
-    testFixturesApi testFixtures(project(':airbyte-cdk:java:airbyte-cdk:core'))
+    testFixturesImplementation project(':airbyte-cdk:java:airbyte-cdk:airbyte-cdk-dependencies')
```

Comment:
so this is the actual gradle convention, which we are messing with artifact name renaming ?

Reply:
Good question! With gradle it's really hard to find documentation about the proper way of doing things. If it's in the gradle manual, great, that's a solid and well-written resource, but for anything else that falls through the cracks it's github issues and forum posts which are often stale because gradle changes all the time.

Artifact renaming is a legit thing in my book, but gradle arguably does a poor job of supporting it properly. At least, that's been my experience as well as that of randos on the internet that showed up in my google searches.